src/terms/Hamiltonian.jl

```diff
@@ -0,0 +1,96 @@
+### A Hamiltonian is composed of blocks (kpoints), which have a list of RFO corresponding to each term
+# This is the "high-level" interface, provided for convenience
+using Printf
```

Comment:
I'd put using / import first

Also I think Printf is `using`ed at global scope in `DFTK.jl` if I am not mistaken.